play-app/src/Context/FilterCont.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { createContext,useContext,useReducer } from "react";
+import { ProdutFilterRed } from '../Reducer/ProductFilterRed';
+
+const FilterContext = createContext();
+
+const ProductFilterProvider = ({children}) =>{
+  const [filterState,filterdispatch] = useReducer(ProdutFilterRed,{
+      sortBy: "",
+      Game : false,
```

Comment:
Suggestion: You can add different data structure for categories in filterstate,  example  
{ sortBy:" ", 
categories:{
      Game : false,
      VR:false,
      ......
       }

Reply:
will try